engine/src/main/java/org/terasology/logic/players/FirstPersonHeldItemMountPointComponent.java

```diff
@@ -50,14 +50,28 @@
             0.0f, -0.05f, -0.2f, 1.0f
     );
 
+    /**
+     * If possible, make this object listen for controller pose upates.
+     */
     public void trySubscribeToControllerPoses() {
         vrProvider.getState().addControllerListener(this);
     }
 
+    /**
+     * A callback target for the controller listener. Not intended to be called manually.
```

Comment:
the method is public by necessity - so that the vrProvider can call it when the listener triggers - but it's not really designed as a "public" method. It's not intended to be part of the public interface.